### ISSUE TITLE
typescript error with sequelize literal

### ISSUE THREAD

**Issue by fareshan** [RobinBuschmann/sequelize-typescript#642]
**Versions**
- sequelize: 5.8.9
- sequelize-typescript: 1.0.0-beta.3
- typescript: 3.3.3

**I'm submitting a ...** 
[ X ] bug report
[ ] feature request

Trying to migrate from:
```
sequelize: 5.0.0-beta.12
sequelize-typescript: 0.6.9
typescript: 2.9.1
```
to:
```
sequelize: 5.8.9
sequelize-typescript: 1.0.0-beta.3
typescript: 3.3.3
```

This is no more working:
```ts
const aas = await A.findAll({
      where: Sequelize.literal(
        `id IN (
            SELECT "aaId"
            FROM "Pp"
            WHERE "ppId" = 10
            )
        `,
      ),
    });
```
The error I get:
```
Argument of type '{ where: Literal; }' is not assignable to parameter of type 'FindOptions'.
  Types of property 'where' are incompatible.
    Type 'Literal' is not assignable to type 'WhereAttributeHash | AndOperator | OrOperator | Where | undefined'.
      Type 'Literal' is not assignable to type 'WhereAttributeHash'.
        Index signature is missing in type 'Literal'.ts(2345)
```

**Comment by RobinBuschmann** [comment]
Hey @fareshan, this also seems to be an issue with the official sequelize typings. So this need to be fixed on sequelize side.

**Comment by fareshan** [comment]
https://github.com/sequelize/sequelize/issues/11080

**Comment by fareshan** [comment]
this issue is fixed with sequelize version 5.9.4
